### PR TITLE
Adding first stale bot to remove everything over 1 year old

### PR DESCRIPTION
**Pull request by nebuk89**
Adding in an action to start removing our old issues so we can see what is going on, initially we will remove everything over 1 year.

### CODE REVIEW

File: .github/workflows/issues_bot.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+name: 'Automatically close stale issues and PRs'
+on:
+  schedule:
+    - cron: '0 0 * * *'
```
Comment (olegburov) [inline .github/workflows/issues_bot.yml:4]: Maybe worth adding a comment, e.g. `Run once a day at midnight`. I always forget cron job syntax 😄

File: .github/workflows/issues_bot.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v3
+        with:
+          stale-issue-message: 'We are clearing up our old issues and your ticket has been open for one year with no activity. Remove stale label or comment or this will be closed in 15 days.'
+          days-before-stale: 365
+          days-before-close: 15
+          exempt-issue-labels: 'enhancement,feature-request,investigating,needs-work,pending-deploytriaged,'
```
Comment (olegburov) [inline .github/workflows/issues_bot.yml:15]: Nit: Can you please a new line?

Screen Shot 2021-04-22 at 9 35 58 AM

**Review by olegburov — APPROVED** [review]
LGTM

File: .github/workflows/issues_bot.yml (hunk 1 above)
Comment (nebuk89) [reply .github/workflows/issues_bot.yml:4]: Added

File: .github/workflows/issues_bot.yml (hunk 2 above)
Comment (nebuk89) [reply .github/workflows/issues_bot.yml]: Fixed!

**Pull request merged**